lexicons/com/atproto/label/subscribeAllLabels.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.label.subscribeAllLabels",
```

Comment:
we changed the repos one to `subscribeRepos` & with the idea that we'll eventually just add ways to filter your subscription with a default of firehose

probably makes sense here as well

Reply:
I made it `subscribeLabels`, though now that I think of it could even just be `subscribe`.